src/main/java/com/schematica/client/renderer/RenderSchematic.java

```diff
@@ -657,24 +733,24 @@ public void renderWorldBorder(final Entity entity, final float partialTicks)
     }
 
     @Override
-    public void drawBlockDamageTexture(final Tessellator tessellator, final WorldRenderer worldRenderer, final Entity entity, final float partialTicks)
+    public void drawBlockDamageTexture(final Tessellator tessellator, final VertexBuffer VertexBuffer, final Entity entity, final float partialTicks)
     {
         // Not needed
     }
 
     @Override
-    public void drawSelectionBox(final EntityPlayer player, final MovingObjectPosition movingObjectPosition, final int p_72731_3_, final float partialTicks)
+    public void drawSelectionBox(final EntityPlayer player, final RayTraceResult movingObjectPosition, final int p_72731_3_, final float partialTicks)
     {
         // Not needed
     }
 
     @Override
-    public void markBlockForUpdate(final BlockPos pos)
+    public void notifyBlockUpdate(World worldIn, BlockPos pos, IBlockState oldState, IBlockState newState, int flags)
     {
         final int x = pos.getX();
         final int y = pos.getY();
         final int z = pos.getZ();
-        markBlocksForUpdate(x - 1, y - 1, z - 1, x + 1, y + 1, z + 1);
+        markBlocksForUpdate(x - 1, y - 1, z - 1, x + 1, y + 1, z + 1, (flags & 8) != 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 8 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)